pyEpiabm/pyEpiabm/tests/test_utility/test_distance_metrics.py

```diff
@@ -0,0 +1,22 @@
+import unittest
+
+from pyEpiabm.utility import DistanceFunctions
+
+
+class TestDistanceFunctions(unittest.TestCase):
+    """Test the 'DistanceFunctions' class.
+    """
+
+    def test_dist_euclid(self):
+        dist = DistanceFunctions.dist_euclid((3, 0))
+        self.assertTrue(dist == 3)
+
+        dist = DistanceFunctions.dist_euclid((-3, 0))
+        self.assertTrue(dist == 3)
+
+        dist = DistanceFunctions.dist_euclid((-3, 0), (3, 0))
+        self.assertTrue(dist == 6)
+
```

Comment:
```suggestion
        dist = DistanceFunctions.dist_euclid((-3, 0), (3, 0))
        self.assertTrue(dist == 6)

        dist = DistanceFunctions.dist_euclid((-2, 0), (1, 4))
        self.assertTrue(dist == 5.0)
```
I think it would be good to have at least one check for the y coordinates as well - this is just a suggestion and I haven't actually ran it to verify it works! Also I think the function should probably return a float?